zendesk/ticket_comment.go

```diff
@@ -20,6 +20,7 @@ type TicketComment struct {
 	AuthorID    int64        `json:"author_id,omitempty"`
 	Attachments []Attachment `json:"attachments,omitempty"`
 	CreatedAt   time.Time    `json:"created_at,omitempty"`
+	Uploads     []string     `json:"uploads"`
```

Comment:
should `omitempty` if it is not mandatory field.

```suggestion
	Uploads     []string     `json:"uploads,omitempty"`
```

Reply:
Agree, changed that. Thanks!